spec/classes/snmp_client_spec.rb

```diff
@@ -36,17 +36,15 @@
           :operatingsystemrelease => '6.4'
         }
         end
-        it { should contain_package('snmp-client').with(
-          :ensure => 'present',
-          :name   => 'net-snmp-utils'
-        )}
+
+        it { should contain_package('snmp-client').that_comes_before('File[snmp.conf]') }
+
```

Comment:
Thanks.  I wanted to use $snmp::params::package_name because the package name may change in future RedHat releases but I could also just set it statically to 'net-snmp'.

I also tried updating the :require => line but that causes the automated tests to fail.  If there's a way to make the spec file accept an array value then I can fix the tests to make it work.